code/modules/clothing/clothing.dm

```diff
@@ -111,6 +111,17 @@
 					LAZYSET(user_vars_remembered, variable, user.vars[variable])
 					user.vv_edit_var(variable, user_vars_to_edit[variable])
 
+/obj/item/clothing/Topic(href, href_list)
+	..()
+	if(href_list["show_valid_pocket_items"])
+		to_chat(usr, "<span class='notice'>[src] can hold: </span>")
+
+		GET_COMPONENT(pockets, /datum/component/storage)

```

Comment:
You should not be using get component calls with components if at all possible, this is a good opportunity to just add a topic signal. Add one at /datum/Topic and make storage listen for it, you can do all this inside the receiver then.

Yes I know this an odd request considering the rest of storage component code but remember that storage components are a prime example of poor component design.